libs/clients/national-registry/v2/src/lib/types/family.dto.ts

```diff
@@ -0,0 +1,21 @@
+import { formatIndividualDto, IndividualDto } from './individual.dto'
+import { Fjolskyldumedlimir } from '../../../gen/fetch'
+
+export interface FamilyDto {
+  familyId: string
+  individuals: IndividualDto[]
+}
+
+export function formatFamilyDto(
+  family: Fjolskyldumedlimir | null,
+): FamilyDto | null {
+  if (family == null || family.fjolskyldunumer == null) {
+    return null
+  }
+  return {
+    familyId: family.fjolskyldunumer,
+    individuals: (family.einstaklingar ?? []).map(
+      (individual) => formatIndividualDto(individual)!,
```

Comment:
Could you use the `is` operator to skip the non-null assertion, if not please add es-lint ignore for the line.